dv_utils/log_utils.py

```diff
@@ -7,6 +7,9 @@
 import sys
 from enum import Enum
 
+from .settings import Settings
+from .settings import settings as default_settings
```

Comment:
As this is the only change to the file, I assume they are not used and thus not needed?

Reply:
✅

Reply:
looks like you forgot to delete this one but it's ok, I will do it